Added a little bit of pathing

Aellantis opened the pull request:
Added some pathing so that the pages link together. Payments page works fine

Concerns:
- Payments page has a back button, but not sure where it should lead to?
- Maintenance page wont work (I think the app/route for maintenance has not been added)
- Profile page does not have nav bar or footer due to not using the `{% extends "index.html" %} {% block content %}`

## Video:
https://github.com/user-attachments/assets/37e1cbcd-5d69-4fdd-a57d-656e45b27b3d

Aellantis committed:
Added a little bit of pathing

xcbhx commented:
Thanks for adding the two additional colors to the landing page. I noticed the video you shared appears to show the same version of your page from the past two weeks, aside from the color updates. Could you clarify if there are any other changes or new implementations?

Regarding the profile page, the route is already set up on my end. Since you weren’t in today’s meeting, I presented my progress to the group. I’ve implemented full CRUD functionality and have had {% extends "index.html" %} in place since the day I walked you through how it works. Currently, I’m working on session handling for the profile page and plan to push those updates by Monday, Feb. 24. I hope to see you in the next meeting.

Let me know if you have any questions.

Aellantis commented:
> Thanks for adding the two additional colors to the landing page. I noticed the video you shared appears to show the same version of your page from the past two weeks, aside from the color updates. Could you clarify if there are any other changes or new implementations?

 I've adjusted the index.html (nav bar) to include pathing. Also went into other files to change a bit of pathing which is this PR. I have not pushed it in case it causes conflicts since I'm going outside of my files ( Killians for example). I've also changed the font which seems to have applied to every other page. Outside of my code. The actual design other than colors has not changed, but we had discussed whether I wanted to change any components, and I liked the minimalistic nature, especially with it's responsiveness should it be converted to mobile view.


> Regarding the profile page, the route is already set up on my end. Since you weren’t in today’s meeting, I presented my progress to the group. I’ve implemented full CRUD functionality and have had {% extends "index.html" %} in place since the day I walked you through how it works. Currently, I’m working on session handling for the profile page and plan to push those updates by Monday, Feb. 24. I hope to see you in the next meeting.

 Before I opened this PR, I made sure to pull the most recent changes from main (hence accessing Killians page and having whatever was last updated).  I'm not sure what happened, but I didn't have the extends code for your page.

Questions: Not sure what Session Handling is. 


I should be able to make the next meeting. I've just been in and out the hospital, so even with joining Dani's class briefly to have it in the background, I was scheduling appts and went to the hospital this afternoon. 

Let me know if any additional clarification is needed.

Aellantis committed:
just fixing a typo

Commit:
coded register and forgot password templates, and its route. redid login route

Commit:
coded register and forgot password templates, and its route. redid login route

Commit:
coded register and forgot password templates, and its route. redid login route

xcbhx committed:
edited session

xcbhx committed:
edited session

xcbhx committed:
removed route & edit route

xcbhx committed:
removed route & edit route

Commit:
update

Aellantis committed:
added maintenance button/section + clarity
